redbot/core/dev_commands.py

```diff
@@ -115,7 +115,26 @@ def sanitize_output(ctx: commands.Context, input_: str) -> str:
 
         result = self.sanitize_output(ctx, str(result))
 
-        await ctx.send(box(result, lang="py"))
+        def check(m):
+            return m.content.lower() == "more"
+
+        pages = list(pagify(result, shorten_by=18))
+        for idx, page in enumerate(pages, 1):
+            if idx != 1:
+                last = await ctx.send(_("There are still {} messages. "
+                                        "Type `more` to continue."
+                                        "").format(len(pages) - (idx - 1)))
+
+                try:
+                    msg = await ctx.bot.wait_for('message', check=check, timeout=10)
+                except asyncio.TimeoutError:
+                    await ctx.send("Your msg timed out")
```

Comment:
I feel like this should instead just delete the "Type `more` to continue." message, since in my opinion that is a good enough indicator that their response timed out, and it doesn't clutter the chat.